commercetools.Sdk/IntegrationTests/commercetools.Sdk.IntegrationTests/ShippingMethods/ShippingMethodIntegrationTests.cs

```diff
@@ -306,19 +306,22 @@ public async Task UpdateShippingMethodChangeName()
         }
 
         [Fact]
-        public async Task UpdateShippingMethodSetDescription()
+        public async Task UpdateShippingMethodSetLocalizedDescription()
```

Comment:
please add an additional test for this method instead of rewriting an existing one